common/pb/session_commands.proto

```diff
@@ -149,6 +150,7 @@ message Command_AccountEdit {
     optional string email = 2;
     // gender = 3; // obsolete
     optional string country = 4;
+    optional string password_check = 100; // password is required to change sensitive information
```

Comment:
Any reason 100?

Reply:
nope, it's not in the way and similar to other places we do this